lib/class-wp-rest-menu-items-controller.php

```diff
@@ -12,6 +12,11 @@
  * @see WP_REST_Posts_Controller
  */
 class WP_REST_Menu_Items_Controller extends WP_REST_Posts_Controller {
+
+	public $validate_order_and_hierarchy = true;
```

Comment:
Isn't it weird to set properties of the controller from some completely different abstraction? The controller should be the top level and be modified via `$request` I think.